pkg/plugins/builtin/ssh/README.md

```diff
@@ -56,11 +56,11 @@ action:
 
 ## Requirements
 
-None by default. Ssh credentials should be retrieved from `{{.config.mySSHKey}}` rather than hardcoded on the template.
+None by default. SSH credentials should be retrieved from `{{.config.mySSHKey}}` rather than hardcoded on the template.
```

Comment:
```suggestion
None. SSH credentials should be retrieved from e.g. `{{.config.mySSHKey.privateKey}}` rather than hardcoded in the template.

Example configuration object __mySSHKey__ (yaml):
``
privateKey: |-
  -----BEGIN OPENSSH PRIVATE KEY-----
  ...
  -----END OPENSSH PRIVATE KEY-----
```